### ISSUE TITLE
Things added to alias do not appear in the text reference section

### ISSUE THREAD

**Issue by Eikosa** [remnoteio/remnote-issues#1129]
Things added to alias do not appear in the text reference section. For example, if I open a document called caffeine, it have to show me the rems containing caffeine the part I have told so far is already working but if I add alias named brown to this document it doesn't appear in text references however, what is written in alias also belongs to this document

**Comment by drkvj** [comment]
Yep, I was going to make a feature request on this myself. It makes total sense to show those text references there as well.
